src/hyperlight_host/src/func/host_functions.rs

```diff
@@ -20,7 +20,6 @@ use std::sync::{Arc, Mutex};
 use hyperlight_common::flatbuffer_wrappers::function_types::{
     ParameterType, ParameterValue, ReturnType,
 };
-use paste::paste;
```

Comment:
Actually–can we remove this dependency now? That would allow us to close #341 !

Reply:
It's still being used in layout.rs, I'll remove it as a follow up if that's ok with you